test/commands/lwc/test/run.nut.ts

```diff
@@ -56,8 +56,8 @@ describe('lightning:lwc:test:run', () => {
     const output = execCmd<RunResult>('force:lightning:lwc:test:run', {
       ensureExitCode: 0,
     }).shellOutput.stderr;
-    expect(output).to.include('Test Suites: 13 passed, 13 total');
-    expect(output).to.include('Tests:       65 passed, 65 total');
+    expect(output).to.match(/Test Suites:\s+\d+ passed,\s+\d+\s+total/);
```

Comment:
nit: Why do we have `\s+\d+ ` with a space here when all the rest have `\s+\d+\s+` with no space?